packages/playground/docs/getting-started-nested.md

```diff
@@ -7,32 +7,92 @@ permalink: /playground/
 
 # Getting Started
 
-The UI5 Web Components carry fundamental UI5 qualities, along with others, to provide enterprise-grade features, Fiori UX, and themeability. The goal is to achieve easy consumption of UI5 controls and to lower the entry barrier to use UI5 controls for own applications. By making the consumption of the full-stack UI5 optional, UI5 Web Components allow you to avoid the complexity of the UI5 rendering stack. In general, the UI5 Web Components are targeting web developers who want to have more flexibility to use just HTML tags or arbitrary JS frameworks.
+The UI5 Web Components share fundamental UI5 qualities with others to provide enterprise-grade features, Fiori UX and themeability. The goal is to achieve an easy consumption of UI5 controls to lower the entry barrier to use UI5 controls for own applications and to avoid complexity of the UI5 rendering-stack by making the consumption of the full-stack UI5 optional. In general, the UI5 Web Components target web developers who want to have more flexibility to use just HTML tags or arbitrary JS frameworks.
+
+UI5 Web Components are shipped as ES6 modules and published on NPM. If you are already using a framework that handles bundling of ES6 modules, skip forward to [Installation](#installation) or check the Tutorials section.
+
+## Creating a project
+
+In case you don't have an existing project with frontend tooling, you can start a new one using [Vite](https://vite.dev). The only prerequisite is to have [Node.js](https://nodejs.org/) installed.
```

Comment:
Somehow I think we should state more clear that we show this with Vite, but it can be done with various tools.

Reply:
improved:
Any ES6 module-capable front-end tooling can be used, but in case you don’t already have a preference, you can start a new project using [Vite]